flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/wordcount/StormWordCountRemoteBySubmitter.java

```diff
@@ -71,7 +71,7 @@ public static void main(final String[] args) throws Exception {
 		// conf.put(Config.NIMBUS_THRIFT_PORT, new Integer(6123));
 
 		// The user jar file must be specified via JVM argument if executed via Java.
-		// => -Dstorm.jar=target/flink-storm-examples-0.9-SNAPSHOT-WordCountStorm.jar
+		// => -Dstorm.jar=target/flink-storm-examples-0.10-SNAPSHOT-WordCountStorm.jar
 		// If bin/flink is used, the jar file is detected automatically.
```

Comment:
This is a bug. I just fixed it. The jar file was renamed to "WordCount-StormTopolgy.jar". There is no version number any more. :)